crates/client/rpc/src/lib.rs

```diff
@@ -1501,7 +1499,17 @@ where
                 messages_sent: messages.into_iter().map(message_conversion).collect(),
                 events: events_converted,
                 execution_result,
-                execution_resources: execution_resources.into(),
+                execution_resources: ExecutionResources {
```

Comment:
what was the issue with using the default value? also, why not have one variable and reuse that for easier readability.

Reply:
Default is not implemented on `starknet_core::types::ExecutionResources`